.github/workflows/ci.yml

```diff
@@ -18,3 +18,20 @@ jobs:
         uses: docker://php:7.3-cli
         with:
           args: ./tools/php-cs-fixer fix --diff-format=udiff --dry-run --show-progress=dots --using-cache=no --verbose
+
+  type-checker:
+    name: Type Checker
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@master
+
+      - name: Update dependencies with composer
+        uses: docker://php:7.3-cli
+        with:
+          args: composer update
+
+      - name: Run vimeo/psalm
+        uses: docker://php:7.3-cli
+        with:
+          args: ./tools/psalm --config=.psalm/static-analysis.xml --no-progress --show-info=false && ./tools/psalm --config=.psalm/config.xml --no-progress --shepherd --stats --show-info=false
```

Comment:
Not sure, I need to investigate if there's a better way to run multiple commands - perhaps as a separate step?